website/client-old/js/controllers/newGroupCtrl.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+/*
+ A controller to manage the New Group page
+ */
+
+angular.module('habitrpg')
+  .controller("NewGroupCtrl", ['$scope', '$window', 'Groups', 'Payments',
```

Comment:
It seems to be that you don't use the create group section in the static. Don't you do a redirect?